woodwork/statistics_utils.py

```diff
@@ -343,6 +343,136 @@ def _get_value_counts(dataframe, ascending=False, top_n=10, dropna=False):
     return val_counts
 
 
+def _calculate_iqr_bounds(series=None, quantiles=None):
+    """Calculates bounds for outlier detection using the 1.5*IQR method.
+
+    Args:
+        series (pd.Series, optional): Data used to calculate first and third quartiles used to calcualte
+            the interquartile range. If present, quartiles will be calculated even if passed in.
```

Comment:
Clarification suggestion...
`If present, any values passed in for quantiles will be replaced with quartiles calculated from this series.`

Reply:
agree that this could be clearer. Here's another suggestion

> If present, any values passed in for quantiles will be ignored. Instead,
            the first and third quartiles will be calculated from this series.

Reply:
Works for me!